google/cloud/completion_queue_test.cc

```diff
@@ -87,7 +87,7 @@ TEST(CompletionQueueTest, TimerCancel) {
 
   using TimerFuture = future<StatusOr<std::chrono::system_clock::time_point>>;
   auto worker = [&](CompletionQueue cq) {
-    for (int i = 0; i != 10000; ++i) {
+    for (int i = 0; i != 1000; ++i) {
```

Comment:
I think we should change the comments on line 81 to reflect the new expected values.  Something like:

```cc
// The magic numbers in this test (number of iterations, thread, etc.) were originally tuned
// to repro #5141 in 99 out of 100 runs using the workstations available at the time.  Since
// then, the number of iterations was reduced. The higher number was making the test too
// long with a the new, more deterministic albeit slower implementation of gRPC alarms.
// The new implementation expires the alarm in a I/O thread vs. sometimes using the thread
// scheduling the alarm.  We expect a repro in 10 out of 100 runs.
```

Reply:
Done, PTAL.